pcdsdevices/interface.py

```diff
@@ -1140,6 +1152,7 @@ def _register_method(self, obj, method_name, method):
         self._methods.append((obj, method_name))
         setattr(obj, method_name, MethodType(method, obj))
         if hasattr(obj, '_tab'):
+            # obj._tab: TabCompletionHelperInstance
```

Comment:
Nitpick: lingering commented code line

Reply:
I kinda thought that would be useful for future us as a sort of pseudo type-hint